INSTALL

```diff
@@ -102,54 +103,11 @@ And you should get traps
 ------------------------
 * Configuring Corosync *
 ------------------------
-The corosync executive will automatically determine cluster membership by
-communicating on a specified multicast address and port.
-
-The directory conf contains the file corosync.conf.example
-
-# Please read the corosync.conf.5 manual page
-totem {
-	version: 2
-	secauth: off
-	threads: 0
-	interface {
-		ringnumber: 0
-		bindnetaddr: 192.168.1.1
-		mcastaddr: 226.94.1.1
-		mcastport: 5405
-	}
-}
-
-logging {
-	fileline: off
-	to_stderr: yes
-	to_file: yes
-	to_syslog: yes
-	logfile: /tmp/corosync.log
-	debug: off
-	timestamp: on
-}
-
-The totem section contains three values.  All three values must be set
-or the corosync executive wll exit with an error.
-
-bindnetaddr specifies the address which the corosync Executive should bind to.
-This address should always end in zero.  If the local interface taffic
-should routed over is 192.168.5.92, set bindnetaddr to 192.168.5.0.
-
-mcastaddr is a multicast address.  The default should work but you may have
-a different network configuration.  Avoid 224.x.x.x because this is a "config"
-multicast address.
-
-mcastport specifies the UDP port number.  It is possible to use the same
-multicast address on a network with the corosync services configured for
-different UDP ports.
-
-The timeout section contains seven values.  This section is not normally used,
-but rather used to override the program defaults for the purposes of fine
-tuning for a given networking/processor combination or for debugging purposes.
-Be careful to use the same timeout values on each of the nodes in the cluster
-or unpredictable results may occur.
+The configuration directory (usually /etc/corosync) contains example
+configuration file (corosync.conf.example). Please copy it as a corosync.conf
+and edit it as needed. At least nodelist section has to be changed
+to contain all cluster nodes. For more information about configuration file
+please read the corosync.conf.5 manual page.
```

Comment:
Some English tidying:

The configuration directory (usually /etc/corosync) contains an example
configuration file (corosync.conf.example). Please copy it as corosync.conf
and edit it as needed. At the very minimum, the nodelist section will have to be changed
to list all cluster nodes. For more information about the configuration file
please read the corosync.conf.5 manual page.